src/factories/fromNodePackageJson.node.ts

```diff
@@ -41,18 +41,19 @@ import { PackageUrlFactory as PlainPackageUrlFactory } from './packageUrl'
  * Node-specific ExternalReferenceFactory.
  */
 export class ExternalReferenceFactory {
-  makeExternalReferences (data: PackageJson): ExternalReference[] {
+  makeExternalReferences (data: PackageJson,  buildSystem: string | null = null): ExternalReference[] {
     const refs: Array<ExternalReference | undefined> = []
 
     try { refs.push(this.makeVcs(data)) } catch { /* pass */ }
     try { refs.push(this.makeHomepage(data)) } catch { /* pass */ }
     try { refs.push(this.makeIssueTracker(data)) } catch { /* pass */ }
+    try { refs.push(this.makeBuildSystem(buildSystem)) } catch { /* pass */ }
```

Comment:
not a responsibility of a builder class that works with `package.json`-like data